03-Custom-Actions-In-Python/05-Custom-Writer.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Is there a reason people might want to use h5py over gsd?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Potentially for more intuitive log storage since HDF5 is designed for multidimensional arrays and nested access, but in general no GSD is good at what it does.